bodhi/server/templates/update.html

```diff
@@ -685,19 +775,14 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
-              % if request.registry.settings.get('ci.required'):
+              % if request.registry.settings.get('greenwave_enabled'):
               <div class="p-b-1">
                 <div>
-                  % if request.registry.settings.get('ci.url'):
-                    <a href="${request.registry.settings.get('ci.url')}" title="More information about CI">
```

Comment:
@ralphbean had requested us to use this ```ci.url``` here. I don't have strong feelings about it, but he wanted to be able to link people to information about the tests. If we do decide that we don't need this anymore, please remove ```ci.url``` from the settings as well (I think this was the only place it was used).